django/contrib/admin/templates/admin/app_list.html

```diff
@@ -0,0 +1,40 @@
+{% load i18n %}
+
+{% if app_list %}
+  {% for app in app_list %}
+    <div class="app-{{ app.app_label }} module{% if app.app_url in request.path %} current-app{% endif %}">
```

Comment:
The use of `request` in the template means the admin now has a dependency on `django.template.context_processors.request`. Technically, this is backwards incompatible. That doesn't particularly bother me, but perhaps should be noted in the release notes. This typically exists from the default settings, so perhaps not a problem in practice.

This should be updated in in the admin docs: https://docs.djangoproject.com/en/dev/ref/contrib/admin/#overview

As well as to the admin checks: https://github.com/django/django/blob/d51e090db2110f016dbca1d794c0d379b3df551b/django/contrib/admin/checks.py#L91-L107

(I discovered this rebasing PR #12752.)

Reply:
Addressed in PR #12906